### ISSUE TITLE
[question] is it reasonable ?

### ISSUE THREAD

**Issue by Eyderoe** [albar965/littlenavmap#1228]
dear Alex,

happy new year and question in translation.

is reasonable that two & appear in one sentence ?

Eyderoe

![image](https://github.com/user-attachments/assets/95607f38-e874-4c07-bc7a-917b468f3426)

**Comment by albar965** [comment]
No, it's not. This is a bug in the original text. There can be only one mnemonic for a menu item, button or other text.
You can translate it as is and omit the duplicate mnemonic in the translation. I can take care of this in the original text.

BTW: Mnemonics (the character after the '&') can be changed to whatever fits best to your language. You don't have to stick to the original text.

Wish you all the best for the New Year! :fireworks:  I still have almost six hours of 2024 left. :upside_down_face: 

Alex

**Comment by Eyderoe** [comment]
yep, but its pity that chinese is not suited for Mnemonics

**Comment by Eyderoe** [comment]
![image](https://github.com/user-attachments/assets/abff09ac-9787-494c-8f1e-8df5517123ab)
and heres another one

**Comment by Eyderoe** [comment]
one old data here (support 10,600 now)
![image](https://github.com/user-attachments/assets/bc5046ec-3caf-42f5-b1ef-bdf86753b6c9)

**Comment by albar965** [comment]
> yep, but its pity that chinese is not suited for Mnemonics

You can leave them out, i.e. remove the '&' from the translated text. This will not hurt any functionality.

> one old data here (support 10,600 now)

Thank you for letting me know. :+1:  Will take care of this too.

Alex

**Comment by Eyderoe** [comment]
1.
something wrong happen in random flight
translation do not work here

![image](https://github.com/user-attachments/assets/a3d77601-8ca9-48f2-b3c8-3e89002272f4)
![image](https://github.com/user-attachments/assets/40a7224e-d792-4cc2-8d21-31fb42db539c)

2.
little crowd here, better a seperate line ?

![image](https://github.com/user-attachments/assets/522d4a68-4fdd-413f-a7ae-86b1975e342c)

**Comment by Eyderoe** [comment]
and here
![image](https://github.com/user-attachments/assets/0800e009-f538-4f18-805c-eae754d3a9fe)

**Comment by albar965** [comment]
All above issues are fixed. The Chinese translation options now also appears in the builds.

**Comment by Eyderoe** [comment]
great thanks
